### PR TITLE
Fix on Lint issues in 'model'

### PR DESCRIPTION
**Pull request by duglin**
There are some TODO's in the code in spots where I'd like to do more investigation into why these types exist - or to see if we can move them from the core code and into the sample code.

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: model/service_broker/service_plan.go [hunk 1]
```diff
@@ -16,7 +16,8 @@ limitations under the License.
 
 package model
 
-//
+// ServicePlan defines on QoS option for a service that a client can
```
Comment (bmelville) [inline model/service_broker/service_plan.go]: Can you clarify this comment? Not really sure what it means exactly, specifically QoS

File: model/service_controller/service_instance.go [hunk 2]
```diff
@@ -20,6 +20,8 @@ import (
 	"k8s.io/client-go/1.5/pkg/runtime"
 )
 
+// ServiceInstanceData is an indirect reference to a service instance.
+// TODO(Dug) - why do we need this?
```
Comment (bmelville) [inline model/service_controller/service_instance.go]: Relic of the old instance-to-instance binding stuff.

File: model/types.go [hunk 3]
```diff
@@ -16,12 +16,14 @@ limitations under the License.
 
 package model
 
+// Types - no idea what this is for - TODO(Dug)
```
Comment (bmelville) [inline model/types.go]: Should be able to drop this whole file I think, this is another relic from the prototype code.

File: model/service_broker/service_plan.go (hunk 1 above)
Comment (duglin) [inline model/service_broker/service_plan.go]: modified - see if its any better

File: model/service_controller/service_instance.go (hunk 2 above)
Comment (duglin) [inline model/service_controller/service_instance.go]: removed

**Comment by duglin** [comment]
I'm ok with lint'ing files that might be removed soon - I'm more interested in being able to add all src dirs into our "verify" makefile target.

**Comment by duglin** [comment]
closing since it appears #176 will cover it

**Pull request closed without merging**